### DISCUSSION TITLE
Send Dicom Images directly to Weasis

### DISCUSSION THREAD

**Discussion by TheLoFang** [nroduit/Weasis#363] [General]
Hello, there is the possibility of being able to send the images directly to weasis. Without importing from the File Menu option?

Send directly from study modalities to weasis?

Thank you everyone

**Comment by nroduit** [comment]
No, because Weasis works in standalone or web mode and not in service mode. Moreover, it does not keep any data when it is closed.

It is not a PACS or a workstation-PACS.

**Reply by TheLoFang** [reply]
> No, because Weasis works in standalone or web mode and not in service mode. Moreover, it does not keep any data when it is closed.
> 
> It is not a PACS or a workstation-PACS.

Thank you very much for the information.
It happens that I am looking for an open source software that allows me to print both in Dicom and in a normal printer. Weasis seemed like an excellent option, but I need the studies to remain in the software for at least 15 to 20 days.

**Reply by tonygonzalez811** [reply]
Greetings, consider using dcm4chee as your image archiving system (PACS) and weasis as your viewer, they work very well together

**Reply by TheLoFang** [reply]
Hi, Tony,
Indeed, I have version 2.18 and dcm4chee installed, I will follow this recommendation with a little more attention, I was really trying to find a totally web weasis viewer, which does not require installation in windows and allows printing, but apparently weasis does not exist totally web, Sometimes it is a bit cumbersome for technicians when they have to use 2 systems, one web, dcm4chee, and the desktop one: weasis.

Grateful
